bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/handler/PulseaudioHandler.java

```diff
@@ -415,28 +368,39 @@ public void setVolume(int volume) {
         savedVolume = volume;
     }
 
-    @Override
-    public void onDeviceStateChanged(ThingUID bridge, AbstractAudioDeviceConfig device) {
-        if (device.getPaName().equals(name)) {
+    public void deviceUpdate(@Nullable AbstractAudioDeviceConfig device) {
+        if (device != null && device.getPaName().equals(name)) {
             updateStatus(ThingStatus.ONLINE);
             logger.debug("Updating states of {} id: {}", device, VOLUME_CHANNEL);
             savedVolume = device.getVolume();
-            updateState(VOLUME_CHANNEL, new PercentType(savedVolume));
+            updateState(VOLUME_CHANNEL, new PercentType(device.getVolume()));
```

Comment:
Why changing that ?
device.getVolume() is put in savedVolume  just the line below

Reply:
savedVolume is nullable, and PercentType don't accept that
Fixed with an additional intermediary variable.

(This is the last comment of your review, I take this opportunity to thank you for it)